charts/fission-all/templates/deployment.yaml

```diff
@@ -151,6 +164,19 @@ spec:
         imagePullPolicy: {{ .Values.pullPolicy }}
         command: ["/fission-bundle"]
         args: ["--routerPort", "8888", "--executorUrl", "http://executor.{{ .Release.Namespace }}"]
+        readinessProbe:
+          httpGet:
+            path: "/router-healthz"
```

Comment:
why not just /healthz?

Reply:
Oh because there might be a function that might want to create a route to `/healthz`. so I didnt want to hog up healthz :) I can change it to `healthz`.